Sources/Foundation/Result/Result+Combine.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Result+Combine.swift
+//  Catalog
+//
+//  Created by Zvonimir Medak on 29.10.2021..
+//  Copyright © 2021 Infinum. All rights reserved.
+//
+
+import Foundation
+
+public extension Result {
+
+    var toCombineResult: Result<Success, Failure> {
+        switch self {
+        case .failure(let error):
+            return .failure(error)
+        case .success(let value):
+            return .success(value)
+        }
+    }
```

Comment:
Could you please remove this var `toCombineResult` @ZvonimirMedak since it doesn't do anything logically.

Reply:
resolved in #54 , nice catch! 🙌